prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -47,13 +47,30 @@ presubmits:
   istio/proxy:
 
   - name: proxy-presubmit
-    context: prow/proxy-presubmit.sh
+    context: "prow/proxy-presubmit"
     branches: *branch_spec
     always_run: true
+    decorate: true
+    path_alias: istio.io/proxy
     labels:
       preset-service-account: "true"
-    spec:
-      <<: *bazel_spec
+    spec: 
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.5.7
+        command:
+        - ./prow/proxy-presubmit.sh
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
```

Comment:
Are you sure? Can you test with phaino? I suspect it will work in a normal container.

Reply:
phaino fails with kind field is not set

Reply:
/hold cancel

Reply:
/hold

Reply:
Tested with removed privileged, it works